stdlib/universe/universe.flux

```diff
@@ -1541,22 +1810,293 @@ builtin quantile : (
+// encountered in the set of table rows is used as the result.
+//
+// The output is constructed as follows:
+//
+// - The set of columns for the new table is the `rowKey` unioned with the group key,
+//   but excluding the columns indicated by the `columnKey` and the `valueColumn`.
+// - A new column is added to the set of columns for each unique value
+//   identified by the `columnKey` parameter.
+// - The label of a new column is the concatenation of the values of `columnKey`
+//   using `_` as a separator. If the value is null, "null" is used.
+// - A new row is created for each unique value identified by the
+//   `rowKey` parameter.
+// - For each new row, values for group key columns stay the same, while values
+//   for new columns are determined from the input tables by the value in
+//   `valueColumn` at the row identified by the `rowKey` values and the new
+//   column’s label. If no value is found, the value is set to `null`.
+// - Any column that is not part of the group key or not specified in the
+//   `rowKey`, `columnKey` and `valueColumn` parameters is dropped.
+//
+// ## Parameters
+// - rowKey: Columns to use to uniquely identify an output row.
+// - columnKey: Columns to use to identify new output columns.
+// - valueColumn: Column to use to populate the value of pivoted `columnKey` columns.
+// - tables: Input data. Default is piped-forward data (`<-`).
+//
+// ## Examples
+//
+// ### Align fields into rows based on time
+// ```
+// # import "csv"
+// #
+// # csvData = "#datatype,string,long,dateTime:RFC3339,string,string,double
+// # #group,false,false,false,true,true,false
+// # #default,,,,,,
+// # ,result,table,_time,_measurement,_field,_value
+// # ,,0,1970-01-01T00:00:01Z,m1,f1,1.0
+// # ,,0,1970-01-01T00:00:01Z,m1,f2,2.0
+// # ,,0,1970-01-01T00:00:01Z,m1,f3,
+// # ,,0,1970-01-01T00:00:02Z,m1,f1,4.0
+// # ,,0,1970-01-01T00:00:02Z,m1,f2,5.0
+// # ,,0,1970-01-01T00:00:02Z,m1,f3,6.0
+// # ,,0,1970-01-01T00:00:03Z,m1,f1,
+// # ,,0,1970-01-01T00:00:03Z,m1,f2,7.0
+// # ,,0,1970-01-01T00:00:04Z,m1,f3,8.0
+// # "
+// #
+// # data = csv.from(csv: csvData)
+// #
+// < data
+// >     |> pivot(rowKey:["_time"], columnKey: ["_field"], valueColumn: "_value")
+// ```
+//
+// ### Associate values to tags by time
+// ```
+// import "sampledata"
+//
+// < sampledata.int()
+// >     |> pivot(rowKey: ["_time"], columnKey: ["tag"], valueColumn: "_value")
+// ```
+//
+// introduced: 0.7.0
+// tags: transformations
+//
 builtin pivot : (<-tables: [A], rowKey: [string], columnKey: [string], valueColumn: string) => [B]
     where
     A: Record,
     B: Record
+
+// range filters rows based on time bounds.
+//
+// Input data must have a `_time` column of type time.
+// Rows with a null value in the `_time` are filtered.
+// Each input table’s group key value is modified to fit within the time bounds.
+// Tables with all rows outside the time bounds are filtered entirely.
```

Comment:
It's probably worth mentioning here that this transformation will add `_start` and `_stop` columns to its output, and also add them to the group key.

Reply:
Good call. Added this information in.